diff --git a/Cargo.toml b/Cargo.toml
index 81563c3..82c5ebd 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -29,6 +29,9 @@ features = ["applet", "tokio", "wayland"]
 version = "0.15"
 features = ["fluent-system", "desktop-requester"]
 
+[patch."https://github.com/smithay/client-toolkit.git"]
+sctk = { package = "smithay-client-toolkit", version = "=0.19.2" }
+
 [package.metadata.deb]
 license-file = "LICENSE"
 assets = [
